src/components/WhatsNew.vue

```diff
@@ -86,39 +85,31 @@
             </li>
             <!-- end blog list item -->
             <!-- blog list item -->
+            <li>
+              <span class="date-text">5/12/2022:</span> The 30 Day Chart Challenge with the USGS Vizlab  
+              <a
+                href="https://waterdata.usgs.gov/blog/chart-challenge-2022/"
+                target="_blank"
+              ><span class="arrow">Read &#8594;</span></a>
+            </li>
+            <!-- end blog list item -->
+            <!-- blog list item -->
             <li>
               <span class="date-text">
-                11/3/2021:
+                11/10/2021:
               </span> 
-              Data viz hiring   
+              What is the USGS Vizlab? 
               <a
-                href="https://waterdata.usgs.gov/blog/viz-hires-2021/"
+                href="https://waterdata.usgs.gov/blog/what-is-vizlab/"
                 target="_blank"
               >
                 <span class="arrow">
                   Read &#8594;
                 </span>
               </a>
             </li>
-          <!-- end blog list item -->
-          <!-- blog list item -->
-            <li>
-              <span class="date-text">5/18/2021:</span> A month of data viz for the #30DayChartChallenge  
-              <a
-                href="https://waterdata.usgs.gov/blog/30daychartchallenge-2021/"
-                target="_blank"
-              ><span class="arrow">Read &#8594;</span></a>
-            </li>
-
-            <li>
-              <span class="date-text">3/29/2021:</span> Recreating the U.S. River Conditions animations in R  
-              <a
-                href="https://waterdata.usgs.gov/blog/build-r-animations/"
-                target="_blank"
-              ><span class="arrow">Read&#8594;</span></a>
-            </li>
+            <!-- end blog list item -->
           </div>
-          <!-- end blog list item -->
```

Comment:
agree with moving this off to make space. do you think it would be useful to have as a scrolling box, or too busy?

Reply:
Hmm I think it depends on the frequency of our blogs. Right now, I think our contributions seem more timely if we only include the last few, but as we up our output a subtle scrolling box or a arrow key navigation box carousel could be good